lib/pure/strutils.nim

```diff
@@ -1150,6 +1150,9 @@ proc parseFloat*(s: string): float {.noSideEffect,
   ##
   ## If `s` is not a valid floating point number, `ValueError` is raised.
   ##``NAN``, ``INF``, ``-INF`` are also supported (case insensitive comparison).
+  ##
+  ## See also:
+  ## * `strmisc <strmisc.html>`_
```

Comment:
`parseFloatThousandSep` should appear in the link; if annoying to type (which I understand, see https://github.com/nim-lang/RFCs/issues/125 for better doc links), just use:
```See also: `strmisc.parseFloatThousandSep` ```